### PR TITLE
feat: video filters on android

### PR DESCRIPTION
**Pull request by santhoshvai**
new package: https://www.npmjs.com/package/@stream-io/video-filters-react-native

## Extra

- the new package had prettier v3 but the workspace was on v2, so I had to update the prettier version of the entire workspace causing a lot of lint updates on other packages and sample apps

### CODE REVIEW

File: packages/react-native-sdk/android/build.gradle
Comment (santhoshvai) [inline packages/react-native-sdk/android/build.gradle:1]: unrelated change, but makes the gradle build on SDK synchronised to the new library template

File: packages/react-native-sdk/src/components/Call/CallControls/index.tsx
Comment (santhoshvai) [inline packages/react-native-sdk/src/components/Call/CallControls/index.tsx:1]: unrelated, its the addition of a missed export

File: sample-apps/react-native/dogfood/src/components/MeetingUI.tsx
Comment (santhoshvai) [inline sample-apps/react-native/dogfood/src/components/MeetingUI.tsx:1]: unrelated, but fixes a bug here on unmount

File: packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/10-apply-video-filters.mdx [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+id: apply-video-filters
+title: Video filters
+description: How to add video filters
+---
+
+In this guide, we will show you how to apply video filters to a video stream. Our SDK offers background blurring and virtual backgrounds out of the box and also has support for injecting your custom filter into the calling experience.
+
+:::note
+Video filters are currently only supported in Android by the SDK.
```
Comment (khushal87) [inline packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/10-apply-video-filters.mdx:10]: May be make **Android** as bold text here.

**Pull request merged**